utils/core_maths/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "core_maths"
+version = "0.0.0"
+description = "Extension trait for full float functionality in `#[no_std]` backed by `libm`."
+
+rust-version.workspace = true
+authors.workspace = true
+edition.workspace = true
+repository.workspace = true
+homepage.workspace = true
+license-file.workspace = true
+categories.workspace = true
+include.workspace = true
+
+[dependencies]
+libm = "0.2"
```

Comment:
should this dep be `cfg(target_os = none)`?

Reply:
Sure

Reply:
actually this doesn't work, it's possible to build `no_std` on a target with an OS.